segments/uptime.sh

```diff
@@ -1,6 +1,29 @@
 # Prints the uptime.
 
+
+if shell_is_bsd; then
+    TMUX_POWERLINE_SEG_UPTIME_GREP_DEFAULT="/usr/local/bin/grep"
```

Comment:
@erikw I think he's assuming that a newer version of (GNU) grep is installed in /usr/local by brew. My grep is in /usr/bin.